### PR TITLE
creating mock data

### PR DESCRIPTION
**Pull request by praktochmalen**

### CODE REVIEW

File: __tests__/localtion/index.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const location = require('../../src/location/index.js')
```
Comment (igorkosta) [inline __tests__/localtion/index.test.js]: @praktochmalen 
- check the name of the folder: `localtion`
- add `'use strict'` at the very top of the file

File: __tests__/localtion/index.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+global.Math.random = () => 0.5
```
Comment (igorkosta) [inline __tests__/localtion/index.test.js]: @praktochmalen what is it good for?

File: __tests__/localtion/index.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+test('show a country', () => {
+  expect(location.country()).toBeTruthy()
```
Comment (igorkosta) [inline __tests__/localtion/index.test.js]: @praktochmalen you should test whether `location.country()` returns a country name from the list of countries, e.g.

```js
const countries = require('../../src/location/countries.js')

test('returns a random country from the list', () => {
  expect(countries).toContain(location.country())
})
```

File: __tests__/localtion/index.test.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+})
+
+test('show a city', () => {
+  expect(location.city()).toBeTruthy()
```
Comment (igorkosta) [inline __tests__/localtion/index.test.js]: @praktochmalen improve the test

File: __tests__/name/first.test.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const first = require('../../src/name/first.js')
+
+test('matches element', () => {
```
Comment (igorkosta) [inline __tests__/name/first.test.js]: @praktochmalen improve the test

File: __tests__/name/index.test.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+const name = require('../../src/name/index.js')
+global.Math.random = () => 0.5
```
Comment (igorkosta) [inline __tests__/name/index.test.js]: @praktochmalen what is it good for?

File: __tests__/name/index.test.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+
+test('show full name when point to full name', () => {
```
Comment (igorkosta) [inline __tests__/name/index.test.js]: @praktochmalen mmm, what does it mean? 😄

File: __tests__/name/index.test.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+  expect(name.full()).toEqual(name.first() + ', ' + name.last())
+})
+
+test('show show an email format', () => {
```
Comment (igorkosta) [inline __tests__/name/index.test.js]: @praktochmalen too many `show`s

File: __tests__/name/last.test.js [hunk 9]
```diff
@@ -0,0 +1,5 @@
+const last = require('../../src/name/last.js')
+
+test('matches element', () => {
```
Comment (igorkosta) [inline __tests__/name/last.test.js]: @praktochmalen doesn't say anything

File: __tests__/name/last.test.js [hunk 10]
```diff
@@ -0,0 +1,5 @@
...
+  expect(last).toBeTruthy()
```
Comment (igorkosta) [inline __tests__/name/last.test.js]: @praktochmalen improve the test

File: __tests__/text/text.test.js [hunk 11]
```diff
@@ -0,0 +1,22 @@
+const text = require('../../src/text/index.js')
+global.Math.random = () => 0.5
```
Comment (igorkosta) [inline __tests__/text/text.test.js]: @praktochmalen why?

File: __tests__/text/text.test.js [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+test('show some random texts', () => {
```
Comment (igorkosta) [inline __tests__/text/text.test.js]: @praktochmalen name tests better, please

File: __tests__/util.test.js [hunk 13]
```diff
@@ -0,0 +1,9 @@
+const rand = require('../src/util.js')
+
+test('it should random array', () => {
+  global.Math.random = () => 0.5 // Math.random will always return 0.5
```
Comment (igorkosta) [inline __tests__/util.test.js]: @praktochmalen why?

File: __tests__/util.test.js [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+  const array = ['666', '2', '123', '69']
+  //  array.length = 4
```
Comment (igorkosta) [inline __tests__/util.test.js]: @praktochmalen remove comments

**Review by igorkosta — COMMENTED** [review]
@praktochmalen please:
- add `'use strict'` 
- check indentation
- improve tests
- remove comments
- don't override `Math.random`

File: __tests__/localtion/index.test.js (hunk 2 above)
Comment (praktochmalen) [reply __tests__/localtion/index.test.js]: @igorkosta Math.random always returns 0.5

File: __tests__/localtion/index.test.js (hunk 3 above)
Comment (praktochmalen) [reply __tests__/localtion/index.test.js]: noted

**Pull request merged**